php/tests/encode_decode_test.php

```diff
@@ -898,6 +898,13 @@ public function testEncodeTopLevelListValue()
         $this->assertSame("[1.5]", $m->serializeToJsonString());
     }
 
+    public function testEncodeEmptyListValue()
+    {
+        $m = new TestMessage();
+        $m->setRepeatedInt32([]);
```

Comment:
This doesn't look like it tests ListValue?

Reply:
Oh, I guess I removed the wrong test. Thanks for the catch.